.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
```

Comment:
Do we need ignore paths? This check takes ~15s, and the problem with limiting actions based on changed paths is that we can't do the same with the protected branch required checks. So if this action sometimes doesn't run, we can't require it.

Reply:
I don't think it is about the time it takes, it is mostly about this linter only makes sense if java code has been changed, hence the ignore.